Classy/Parser/CASStyler.h

```diff
@@ -12,7 +12,7 @@
 
 @interface CASStyler : NSObject
 
-+ (void)bootstrapClassy;
++ (void)bootstrapClassyWithTargetWindows:(NSArray *)targetWindows;
```

Comment:
@killobatt Why was this change made, and how should one bootstrap Classy now? IE., what windows are to be provided when calling this?

cc: @mikeger